### PR TITLE
Add media icons

### PR DESCRIPTION
**Pull request by vitorflg**
#### What problem is this solving?

Add media icons for video block

#### How to test it?

The icons are being used in the video block in the following workspace:

https://vitorflg--storecomponents.myvtex.com/

### CODE REVIEW

File: store/interfaces.json [hunk 1]
```diff
@@ -88,5 +88,17 @@
...
     "component": "IconExpand"
+  },
+  "icon-volume-on": {
+    "component": "IconVolumeOn"
+  },
+  "icon-volume-off": {
+    "component": "IconVolumeOff"
+  },
+  "icon-play": {
+    "component": ",IconPlay"
```
Comment (Klynger) [inline store/interfaces.json]:
```suggestion
    "component": "IconPlay"
```

**Review by Klynger — CHANGES_REQUESTED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -225,6 +232,11 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 | 'socialIcon'         |
 | 'starIcon'           |
 | 'swapIcon'           |
+| 'playIcon'           |
+| 'pauseIcon'          |
+| 'FullscreenIcon'     |
+| 'VolumeOnIcon'       |
+| 'VolumeOffIcon'      |
```
Comment (klzns) [inline docs/README.md]: Can we change to use camelCase?

File: styles/iconpacks/iconpack.svg [hunk 3]
```diff
@@ -29,9 +29,19 @@
       <path d="M11 7H9V13H11V7Z" fill="currentColor"/>
       <path d="M12 1C12 0.4 11.6 0 11 0H5C4.4 0 4 0.4 4 1V3H0V5H1V15C1 15.6 1.4 16 2 16H14C14.6 16 15 15.6 15 15V5H16V3H12V1ZM6 2H10V3H6V2ZM13 5V14H3V5H13Z" fill="currentColor"/>
     </g>
-    <g id="hpa-play">
+    <g id="mpa-play">
       <path d="M3.5241 0.937906C2.85783 0.527894 2 1.00724 2 1.78956V14.2104C2 14.9928 2.85783 15.4721 3.5241 15.0621L13.6161 8.85166C14.2506 8.46117 14.2506 7.53883 13.6161 7.14834L3.5241 0.937906Z" fill="currentColor"/>
     </g>
+    <g id="mpa-pause">
+      <rect width="6" height="16" fill="white"/>
```
Comment (klzns) [inline styles/iconpacks/iconpack.svg]: Change colors to `currentColor`
```suggestion
      <rect width="6" height="16" fill="currentColor"/>
```

**Review by klzns — CHANGES_REQUESTED** [review]

File: styles/iconpacks/iconpack.svg [hunk 4]
```diff
@@ -108,7 +108,7 @@
     <g id="mpa-clone--filled">
       <path d="M6.07501 4.925L6.07501 2.175C6.07501 1.62271 6.52273 1.175 7.07501 1.175H13.825C14.3773 1.175 14.825 1.62271 14.825 2.175V8.925C14.825 9.47729 14.3773 9.925 13.825 9.925H11.075" fill="currentColor"/>
       <path d="M6.07501 4.925L6.07501 2.175C6.07501 1.62271 6.52273 1.175 7.07501 1.175H13.825C14.3773 1.175 14.825 1.62271 14.825 2.175V8.925C14.825 9.47729 14.3773 9.925 13.825 9.925H11.075" stroke="currentColor" stroke-width="2" stroke-miterlimit="10" stroke-linecap="square"/>
-      <path d="M1 6.49999L1 13C1 14.1046 1.89543 15 3 15H9.49997C10.6045 15 11.5 14.1046 11.5 13V6.49999C11.5 5.39542 10.6045 4.49999 9.49997 4.49999H3C1.89543 4.49999 1 5.39542 1 6.49999Z" fill="currentColor" stroke="white" stroke-width="2" stroke-miterlimit="10" stroke-linecap="square"/>
+      <path d="M1 6.49999L1 13C1 14.1046 1.89543 15 3 15H9.49997C10.6045 15 11.5 14.1046 11.5 13V6.49999C11.5 5.39542 10.6045 4.49999 9.49997 4.49999H3C1.89543 4.49999 1 5.39542 1 6.49999Z" fill="currentColor" stroke="currentColor" stroke-width="2" stroke-miterlimit="10" stroke-linecap="square"/>
```
Comment (klzns) [inline styles/iconpacks/iconpack.svg]: I would keep this one as it was
```suggestion
      <path d="M1 6.49999L1 13C1 14.1046 1.89543 15 3 15H9.49997C10.6045 15 11.5 14.1046 11.5 13V6.49999C11.5 5.39542 10.6045 4.49999 9.49997 4.49999H3C1.89543 4.49999 1 5.39542 1 6.49999Z" fill="currentColor" stroke="white" stroke-width="2" stroke-miterlimit="10" stroke-linecap="square"/>
```

**Review by klzns — CHANGES_REQUESTED** [review]
Also, rebase and push force branch

**Review by kaisermann — APPROVED** [review]

File: react/IconVolumeOff.tsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+import Icon from './components/Icon'
+
+const IconVolumeOff = (props: IconProps) => {
+  return <Icon id="sti-volume-off" handle="VolumeOffIcon" {...props} />
```
Comment (klzns) [inline react/IconVolumeOff.tsx]:
```suggestion
  return <Icon id="sti-volume-off" handle="volumeOffIcon" {...props} />
```

**Review by klzns — CHANGES_REQUESTED** [review]

**Review by klzns — APPROVED** [review]

**Review by icazevedo — APPROVED** [review]

**Pull request merged**